assets/src/scripts/outputs-viewer/hooks/use-file.js

```diff
@@ -1,24 +1,27 @@
 import { useQuery } from "react-query";
+import useStore from "../stores/use-store";
 import handleErrors from "../utils/fetch-handle-errors";
 import { canDisplay } from "../utils/file-type-match";
 
-function useFile(authToken, file) {
+function useFile(file) {
+  const { authToken = "" } = useStore();
+
   return useQuery(["FILE", file.url], async () => {
     // If we can't display the file type
     // or the file size is too large
     // don't try to return the data
     if (!canDisplay(file) || file.size > 20000000) return {};
 
-    let config = {};
-    if (authToken) {
-      config = {
-        headers: new Headers({
-          Authorization: authToken,
-        }),
-      };
-    }
-
-    return fetch(file.url, config)
+    return fetch(
+      file.url,
+      authToken
+        ? {
+            headers: new Headers({
+              Authorization: authToken,
+            }),
+          }
+        : null
```

Comment:
I'm not sure we need this conditional.  I had the same idea with conditionally populating `config` originally, and decided to just just always set it with either a value or empty string in useFileList (and then forgot about useFile, oops!), and I don't see an issue with sending the extra header on the request for a bit less code!

Reply:
Cool - I left that in there incase you were handling it in a special way on the server-side.

I've removed it and set up headers for both fetch requests.